src/UserRepo.js

```diff
@@ -5,18 +5,19 @@ import User from './User';
 class UserRepo extends Repo  {
   constructor(usersData) {
     super(usersData)
+    this.currentUser = null;
   }
 
-  storeData(data) {
-    if (Array.isArray(data)) {
-      this.data = data;
-      // this.findCurrentUser()
-    }
-  }
-
+  // storeData(data) {
+  //   if (Array.isArray(data)) {
+  //     this.data = data;
+  //     // this.findCurrentUser()
+  //   }
+  // }
+/* */
   findCurrentUser() {
-    let index = this.GetRandomNumber()
-    let user = this.data.find(user => user.userId === index)
+    let index = this.getRandomNumber()
+    let user = this.data.find(user => user.id === index)
     this.currentUser = new User(user)
```

Comment:
You can actually declare properties outside of the constructor like this, they don't necessarily need a place to land.

Reply:
I couldn't get this to operate correctly without this, so we'll pair tomorrow when we can so we can talk through this/figure out a solution.